package.json

```diff
@@ -18,5 +18,6 @@
   "packageManager": "yarn@4.0.2",
   "devDependencies": {
     "prettier": "^3.2.0"
-  }
+  },
+  "version": "0.0.0"
```

Comment:
I'm not sure where this commit came from.

once I get the 1st report from CI, I'll get rid of it.